src/services/snapshot-service.ts

```diff
@@ -23,27 +23,27 @@ export default class SnapshotService extends PercyClientService {
     this.assetDiscoveryService = new AssetDiscoveryService(buildId, configuration)
   }
 
-  async buildResources(
+  buildResources(
     rootResourceUrl: string,
     domSnapshot = '',
     options: SnapshotOptions,
     logger: any,
   ): Promise<any[]> {
-    const rootResource = this.percyClient.makeResource({
-      resourceUrl: rootResourceUrl,
-      content: domSnapshot,
-      isRoot: true,
-      mimetype: 'text/html',
-    })
-
-    const discoveredResources = await this.assetDiscoveryService.discoverResources(
+    return this.assetDiscoveryService.discoverResources(
       rootResourceUrl,
       domSnapshot,
       options,
       logger,
     )
+  }
```

Comment:
This method no longer seems useful. Could leave it as an alias, or just call `this.snapshotService.assetDiscoveryService.discoverResources` directly from the agent service.

Reply:
Yeah for sure, going to clean that up in future PRs